calico_containers/pycalico/datastore.py

```diff
@@ -134,7 +134,15 @@ class DatastoreClient(object):
 
     def __init__(self):
         etcd_authority = os.getenv(ETCD_AUTHORITY_ENV, ETCD_AUTHORITY_DEFAULT)
-        (host, port) = etcd_authority.split(":", 1)
+        try:
+            (host, port) = etcd_authority.split(":", 1)
```

Comment:
Rather, it will likely catch the error below but the error message won't be great.